### PR TITLE
[JFG-994] workload.threads.maxcount is ignored in java builders configuration

### PR DESCRIPTION
**Pull request by SokolAndrey**

### CODE REVIEW

File: chassis/configuration/configuration/master/master.conf.xml [hunk 1]
```diff
@@ -175,6 +175,7 @@
             </bean>
         </property>
         <property name="limitSetConfig" ref="limitSetConfig"/>
+        <property name="defaultMaxLoadThreads" value="${workload.threads.maxcount:4000}"/>
```
Comment (anton-antonenko) [inline chassis/configuration/configuration/master/master.conf.xml]: set it to 500, please

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileRps.java [hunk 2]
```diff
@@ -68,29 +68,35 @@ public static Builder builder(RequestsPerSecond requestsPerSecond) {
...
+         *
          * @return JLoadProfileRps object.
          */
         public JLoadProfileRps build() {
             return new JLoadProfileRps(this);
         }
 
-        /** Optional: Max load threads. Default is 500.
+        /**
+         * Optional: Max load threads. Default is 4000.
```
Comment (anton-antonenko) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileRps.java]: default must be 500

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileTps.java [hunk 3]
```diff
@@ -78,7 +80,7 @@ public JLoadProfileTps build() {
             return new JLoadProfileTps(this);
         }
 
-        /** Optional: Max load threads. Default is 500.
+        /** Optional: Max load threads. Default is 4000.
```
Comment (anton-antonenko) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileTps.java]: default must be 500

**Review by anton-antonenko — APPROVED** [review]

**Comment by SokolAndrey** [comment]
Btw, this branch still can be merged :smirk:

**Comment by dlatnikov** [comment]
would like to see a demo before merge :)

**Pull request closed without merging**